src/model/element/entity.js

```diff
@@ -165,6 +165,25 @@ class Entity extends Element {
     return update;
   }
 
+  normalized(){
+    // normalized := has a name and is associated
+    const name = this.syncher.get('name');
+    return name && this.associated() && this.completed();
```

Comment:
Not sure what 'completed' was supposed to indicate?

Reply:
associated true (entity/node only) => grounded

completed (any element / node / edge) => the user has finished specifying the element

E.g. an interaction doesn't have a grounding but it can still be completed by specifying it as phosphorylation.